CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ if (BUILD_EXAMPLES)
   if (UNIX)
       add_subdirectory(examples/master)
   endif()
-  if (NUTTX)
+  if (NUTTX AND (NUTTX_ARCH_CHIP STREQUAL "xmc4"))
     add_subdirectory(examples/slave)
```

Comment:
rename slave to have slave_xmc4800?
It seems that if(NUTTX) is useless since is is always defined if NUTTX_ARCH_CHIP is defined.

Reply:
Hmm I moved this condition into the slave/CMakeLists.txt

We also have an example of NuttX using easycat, with the only dependency being SPI driver of NuttX.